### PR TITLE
Speed up service-environment sync from ralph

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: src/ralph_scrooge/utils/common.py [hunk 1]
```diff
@@ -146,3 +153,32 @@ def camel_case_to_kebab_case(name):
     """
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1-\2', name)
     return re.sub('([a-z0-9])([A-Z])', r'\1-\2', s1).lower()
+
+
+class timeit(object):
```
Comment (ar4s) [inline src/ralph_scrooge/utils/common.py:158]: where do you use this class?

Comment (mkurek) [reply src/ralph_scrooge/utils/common.py:158]: currently nowhere, but it's really helpful in tracking down bottlenecks (both time and db queries count) in plugins

File: src/ralph_scrooge/utils/common.py [hunk 2]
```diff
@@ -6,14 +6,21 @@
...
+import logging
 import re
+import time
 from datetime import datetime
 from decimal import Decimal
 
+from functools import wraps
+
 from django.conf import settings
 # DEPRECATED. Import memoize directly from `ralph_scrooge.cache`
```
Comment (ar4s) [inline src/ralph_scrooge/utils/common.py]: This comment is related to 20 line not to 19. It is necessary?

Comment (mkurek) [reply src/ralph_scrooge/utils/common.py]: moved one line below; yes - it's still relevant - there are a lot of imports from `from ralph_scrooge.utils.common import memoize`

**Review by ar4s — APPROVED** [review]

**Review by MarekBleschke — APPROVED** [review]

**Pull request merged**